mapper/drugbank_mapper.tsv

```diff
@@ -0,0 +1,8926 @@
+drugbank_id	chebi_id	MeSH
+DB00035	4450	D003894
```

Comment:
I think we want this file to have only two columns: `drugbank_id` and `mesh_id`. No duplicate rows, no missing values, and sorted.